jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -252,7 +252,7 @@ public Response put(URL url, Request request) throws IOException {
   public Response call(String httpMethod, URL url, Request request) throws IOException {
     if (!isHttpsProtocol(url)) {
       if (enableHttpAndInsecureFailover) { // HTTP requested. We only care if HTTP is enabled.
-        return call(httpMethod, url, request, getHttpTransport(true));
+        return call(httpMethod, url, request, getHttpTransport(true), true);
```

Comment:
This is a terminal call to connect to plain HTTP (i.e., no insecure failover necessary in response to `SSLException`) , so we can enable retrying (`true` for the last argument).